.test-infra/jenkins/job_Inventory.groovy

```diff
@@ -30,7 +30,7 @@ nums.each {
     commonJobProperties.setTopLevelMainJobProperties(delegate)
 
     // Sets that this is a cron job.
-    commonJobProperties.setCronJob(delegate, '45 18 * * *')
+    commonJobProperties.setCronJob(delegate, '0 */12 * * *')
```

Comment:
Should we instead fo with 6,18 to control when the it will run? There might be value in running the inventory task about the same time for all machines.

Reply:
Ack. Done.